run_model.py

```diff
@@ -20,63 +17,76 @@ def main():
 @main.command("county")
 @click.option("--state", "-s")
 @click.option(
-    "--deploy",
-    is_flag=True,
-    help="Output data files to public data directory in local covid-projections.",
+    "--output",
+    "-o",
+    help="Output directory",
+    type=pathlib.Path,
+    default=pathlib.Path("results/county"),
 )
 @click.option(
-    "--summary-only",
-    is_flag=True,
-    help="Only runs the county summary if true.",
+    "--summary-only", is_flag=True, help="Only runs the county summary if true.",
 )
 @data_version.with_git_version_click_option
-def run_county(version: data_version.DataVersion, state=None, deploy=False, summary_only=False):
+def run_county(
+    version: data_version.DataVersion, output, state=None, summary_only=False
+):
     """Run county level model."""
     min_date = datetime.datetime(2020, 3, 7)
     max_date = datetime.datetime(2020, 7, 6)
 
-    output_dir = build_params.OUTPUT_DIR
-    if deploy:
-        output_dir = WEB_DEPLOY_PATH
+    run.run_county_level_forecast(
+        min_date, max_date, output, country="USA", state=state
+    )
+    if not state:
```

Comment:
Missing `summary_only` check. Is there a reason we want to remove it?

Reply:
Oh, i didn't properly clean that up.  Basically the summary is moved to a different command to make output directories more configurable.